go/libraries/doltcore/sqle/database.go

```diff
@@ -321,6 +321,14 @@ func (db Database) getTableInsensitive(ctx *sql.Context, head *doltdb.Commit, ds
 	//  at runtime
 	switch {
 	case strings.HasPrefix(lwrName, doltdb.DoltDiffTablePrefix):
+		if resolve.UseSearchPath && db.schemaName == "" {
+			schemaName, err := resolve.FirstExistingSchemaOnSearchPath(ctx, root)
+			if err != nil {
+				return nil, false, err
+			}
+			db.schemaName = schemaName
```

Comment:
This shouldn't matter because it's a struct receiver, but probably clearer to declare a `doltdb.TableName` var above this block with `db.schemaName`, then fill the schema name in here. Then you can pass it below.

Same comment in other cases.